docs/source/notebooks/subset.ipynb

```diff
@@ -18,10 +18,10 @@
    "id": "8f4fd0c3",
```

Comment:
<p>Is it possible to silence this kind of warnings "/opt/miniconda3/envs/finch/lib/python3.9/site-packages/geopandas/io/file.py:362: FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead."</p><p> </p><p>This will break Jenkins.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bird-house/finch/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Not directly, this has to be done in `geopandas`. but we could filter out any "FutureWarning" I guess.